index.js

```diff
@@ -749,12 +746,22 @@ RedisClient.prototype.send_command = function (command, args, callback) {
         args = args.slice(0, -1).concat(args[args.length - 1]);
     }
 
-    // if the value is undefined or null and command is set or setx, need not to send message to redis
-    if (command === 'set' || command === 'setex') {
-        if(args[args.length - 1] === undefined || args[args.length - 1] === null) {
-            var err = new Error('send_command: ' + command + ' value must not be undefined or null');
-            return callback && callback(err);
-        }
+    // if the value or the key is undefined or null and command is set, setex or mset; need not to send message to redis
+    if (command === 'set' || command === 'setex' || command === 'mset') {
```

Comment:
my preference would be to change the tests, rather than making this breaking change -- perhaps we could update the tests to reflect the actual behavior?